bugzilla/harmony
# 1879461: Runtime errors when building documentation

#### Details

This PR fixes 1879461

#### Additional info
* [bmo#](https://bugzilla.mozilla.org/show_bug.cgi?id=1879461)

#### Test Plan

1. Build documentation with ` sphinx-build -b html -d ./doctrees   en/rst ./html`
2. Confirm runtime error in bugzilla report does not occur
